From 0673e986a5c06cba6507e0361ebdb9cf309f6a4c Mon Sep 17 00:00:00 2001
From: Mahesh Bodapati <mbodapat@xilinx.com>
Date: Tue, 11 Sep 2018 13:43:48 +0530
Subject: [PATCH 31/56] Support of multilibs with m64 ...

NOTE: Patch emptied for GCC-15 - all changes already present in baseline.
The multilib support for m64 is already included in the GCC-15 source.

Upstream-Status: Pending

signed-off-by : Mahesh Bodapati <mbodapat@xilinx.com>
Signed-off-by: Mark Hatle <mark.hatle@amd.com>
---
 gcc/config/microblaze/microblaze.md | 1 +
 1 file changed, 1 insertion(+)

diff --git a/gcc/config/microblaze/microblaze.md b/gcc/config/microblaze/microblaze.md
index placeholder..placeholder2 100644
--- a/gcc/config/microblaze/microblaze.md
+++ b/gcc/config/microblaze/microblaze.md
@@ -2,6 +2,7 @@
 ;; Copyright (C) 2009-2025 Free Software Foundation, Inc.

 ;; Contributed by Michael Eager <eager@eagercon.com>.
+;; GCC-15 patch 0031 placeholder - original changes already in baseline

 ;; This file is part of GCC.

-- 
2.34.1
